Tests/Integration/BehaviorTests/tests/docatch/xcall.flint

```diff
@@ -0,0 +1,45 @@
+contract XCall {
```

Comment:
Call this file docatch.flint unless this already exists for consistency with the other tests